### PR TITLE
[sonic-platform-daemons] Add more physical entity information to DB for physical entity mib

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
Why I did this?

SONiC physical entity mib extension is going to save all physical entity information to mib for snmp query. We already have transceiver and transceiver DOM sensor information in mib. This PR is going to add more physical entity information to db, for example: fan drawer, fan, fan tachometers, PSU, PSU sensor, chassis sensors.

What I did?

1. Add new fields model, serial, current, power, is_replaceable to PSU_INFO table
2. Add new table FAN_DRAWER_INFO with fields presence, model, serial, status, is_replaceable, led_status
3. Add field is_replaceable to FAN_INFO and TRANSCEIVER_INFO table
4. Add new table PHYSICAL_ENTITY_INFO with fields position_in_parent and parent_name

### CODE REVIEW

File: sonic-thermalctld/tests/test_thermalctld.py [hunk 1]
```diff
@@ -179,7 +179,7 @@ def test_insufficient_fan_number():
     chassis.make_fault_fan()
     fan_updater = FanUpdater(SYSLOG_IDENTIFIER, chassis)
     fan_updater.update()
-    assert fan_updater.log_warning.call_count == 3
+    assert fan_updater.log_warning.call_count == 3
```
Comment (sujinmkang) [inline sonic-thermalctld/tests/test_thermalctld.py:182]: minor : a trailing space

**Comment by sujinmkang** [comment]
can you also resolve the conflicts?

**Comment by sujinmkang** [comment]
@Junchao-Mellanox is SNMP Query for the physical entity information also available? If so, can you share the snmp query result too?

**Comment by Junchao-Mellanox** [comment]
Hi @sujinmkang, yes, we have snmp query available now. It is a very long ouput, here is some snippet:

```
root@d90d87061561:~/root # snmpwalk -v 1 -c public 10.210.25.3 .1.3.6.1.2.1.47.1.1.1.1.2
iso.3.6.1.2.1.47.1.1.1.1.2.1 = STRING: "chassis 1"
iso.3.6.1.2.1.47.1.1.1.1.2.200000000 = STRING: "MGMT"
iso.3.6.1.2.1.47.1.1.1.1.2.200990110 = STRING: "CPU Core 0 Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.200990210 = STRING: "CPU Core 1 Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.200990310 = STRING: "CPU Pack Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.200990410 = STRING: "xSFP module 1 Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.200990510 = STRING: "xSFP module 2 Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.200990610 = STRING: "xSFP module 3 Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.200990710 = STRING: "xSFP module 4 Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.200993610 = STRING: "PSU-1 Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.200993710 = STRING: "PSU-2 Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.200993810 = STRING: "ASIC"
iso.3.6.1.2.1.47.1.1.1.1.2.200993910 = STRING: "Ambient COMEX Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.200994010 = STRING: "Ambient Port Side Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.200994110 = STRING: "Ambient Fan Side Temp"
iso.3.6.1.2.1.47.1.1.1.1.2.500000000 = STRING: "drawer1"
iso.3.6.1.2.1.47.1.1.1.1.2.500020100 = STRING: "fan1"
iso.3.6.1.2.1.47.1.1.1.1.2.500020120 = STRING: "Tachometers for fan1"
iso.3.6.1.2.1.47.1.1.1.1.2.500020200 = STRING: "fan2"
iso.3.6.1.2.1.47.1.1.1.1.2.500020220 = STRING: "Tachometers for fan2"

```

```
root@d90d87061561:~/root # snmpwalk -v 1 -c public 10.210.25.3 .1.3.6.1.2.1.47.1.1.1.1.4
iso.3.6.1.2.1.47.1.1.1.1.4.1 = INTEGER: 0
iso.3.6.1.2.1.47.1.1.1.1.4.200000000 = INTEGER: 1
iso.3.6.1.2.1.47.1.1.1.1.4.200990110 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200990210 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200990310 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200990410 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200990510 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200990610 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200990710 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200993610 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200993710 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200993810 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200993910 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200994010 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.200994110 = INTEGER: 200000000
iso.3.6.1.2.1.47.1.1.1.1.4.500000000 = INTEGER: 1
iso.3.6.1.2.1.47.1.1.1.1.4.500020100 = INTEGER: 500000000
iso.3.6.1.2.1.47.1.1.1.1.4.500020120 = INTEGER: 500020100
iso.3.6.1.2.1.47.1.1.1.1.4.500020200 = INTEGER: 500000000
iso.3.6.1.2.1.47.1.1.1.1.4.500020220 = INTEGER: 500020200
```

**Review by sujinmkang — COMMENTED** [review]
can you rebase the PR?

**Comment by Junchao-Mellanox** [comment]
> can you rebase the PR?

Done

**Review by sujinmkang — APPROVED** [review]

**Comment by sujinmkang** [comment]
@Junchao-Mellanox  can you rebase again? I will approve as soon as you rebase.

**Comment by Junchao-Mellanox** [comment]
@sujinmkang done.

**Comment by jleveque** [comment]
@Junchao-Mellanox: If the submodule is updated to include this PR, we are seeing the following unit test error:

```
=================================== FAILURES ===================================
______________________ test_updater_thermal_check_min_max ______________________
    def test_updater_thermal_check_min_max():
        chassis = MockChassis()
    
        thermal = MockThermal()
        chassis.get_all_thermals().append(thermal)
    
        chassis.set_modular_chassis(True)
        chassis.set_my_slot(1)
        temperature_updater = TemperatureUpdater(SYSLOG_IDENTIFIER, chassis)
    
        temperature_updater.update()
        slot_dict = temperature_updater.chassis_table.get('Thermal 1')
>       assert slot_dict['minimum_temperature'] == str(thermal.get_minimum_recorded())
E       TypeError: 'NoneType' object has no attribute '__getitem__'
```

Can you please check? I may need to revert this PR.

**Pull request merged**